### ISSUE TITLE
AIOC keeps PTT open for over 3 sec

### ISSUE THREAD

**Issue by WROG208** [skuep/AIOC#81]
I ordered some a while back and would use them on allstar and gmrslive. GMRSlive is an old version of Allstar repurposed to be used with GMRS, which is just for those who don't know.
I flashed the 1.3 firmware to all boards I used the how-to from G1LRO that had modified the VCOS trail time from 2000 to 1500. I have verified this by looking at the Python script he did.

Well, the problem is that not every board will leave the tail on 1500ms. Some are going to a 3-second delay before the PTT is finally disengaged. As I said it only happens on probably 1 out of 3 cards. I flashed the firmware again to the ones I still have at the house but I can't recreate the problem a vast majority of people I sold the AIOC to are now having the problem.

Anything you guys might need in the form of files and settings let me know and I will make them available. I just want to fix this problem for all these guys.

**Comment by skuep** [comment]
Can you verify if this issue is maybe related to RF interference? What's your antenna setup? We have had some people that had similar issues and there is a (closed?) issue about that with some tips.

**Comment by WROG208** [comment]
![20240714_014852.jpg](https://github.com/user-attachments/assets/48a91ea5-edb6-4558-a0db-9afcd706ef23)

As you can see instead of extending the antenna i extended the AIOC. 30 units were made this way but only a handful has the problem. I personally don't have the problem and no matter what I do i cant replicate it.

**Comment by skuep** [comment]
Interesting! Any specific radio where this happens? 
Did you get them from JLCPCB or somewhere else?

**Comment by WROG208** [comment]
JLCPCB ut has happened with Baofeng UV5R and one with a Baofeng 888.
To be honest i think the problem is more of radio programming than the actual AIOC. I have tried different squelch channels anything i could think of. I even solder one AIOC with the plugs directly on the AIOC and i could not replicate either the problem or the AIOC lok8ng out from RF. The only time I had the problem was at the start with a quansheng uvk5 and didn't think much about it and switched radios to a Baofeng and it never did it again. I even tested a anytone 800 something plus and it worked with no problem and on purpose I set the radio to turbo instead of high power and still no problem.
I programmed a Motorola 7550 on high power with the channel used in the radio with the AIOC nothing worked flawlessly.
